### DISCUSSION TITLE
Sales & Support Non-Responsive for a Week - Please Assist

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142159] [Enterprise]
### Select Topic Area

Question

### Body

We have submitted several requests to both the enterprise sales team and the support team to upgrade our organization to an enterprise with personal accounts, after an issue with creating the managed version and being unable to switch. No one from any team has attempted to reach out to offer assistance or begin the sales process.

**Comment by queenofcorgis** [comment]
Thank you for raising this situation. Myself and my team took a look into this - I believe someone is in contact with your team from sales. 

As for your ticket, we can't guarantee a timeline for a response as it depends on our team's resources, the amount of tickets ahead of yours, and the severity of your issue. 

Appreciate your patience!

**Comment by datonefeed** [comment]
response1
